packages/citrix-cpx-istio-sidecar-injector/generated-changes/patch/Chart.yaml.patch

```diff
@@ -0,0 +1,25 @@
+--- charts-original/Chart.yaml
++++ charts/Chart.yaml
+@@ -1,15 +1,21 @@
++annotations:
++  catalog.cattle.io/certified: partner
++  catalog.cattle.io/display-name: citrix-cpx-istio-sidecar-injector
```

Comment:
Please refer to this comment https://github.com/rancher/partner-charts/pull/167#discussion_r714371797

Reply:
Hi @PennyScissors 

I have a small query here, Is that we can update the display-name in the same PR,  Earlier there was some limitation, we can do max 2 commits in a single PR?

Thanks,
Ajeeta